spec/spec_helper.rb

```diff
@@ -34,3 +35,7 @@ def load_schema
 
 # example model classes
 Dir[File.dirname(__FILE__) + "/models/*.rb"].sort.each { |f| require File.expand_path(f) }
+
+I18n.load_path << 'spec/en.yml'
```

Comment:
The order of loading of `spec/en.yml` is important... These line must be execute after loading of `activerecord` gem because of [this line](https://github.com/aasm/aasm/pull/689/files#diff-91a325edd28df1a02e0ffe9f85ac31cfL12) which overrides default error message of `activerecord`.